tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_get_head.py

```diff
@@ -261,3 +267,67 @@ def test_filtered_block_tree(spec, state):
     })
 
     yield 'steps', test_steps
+
+
+@with_all_phases
+@spec_state_test
+def test_proposer_score_boost_basic(spec, state):
+    test_steps = []
+    genesis_state = state.copy()
+
+    # Initialization
+    store, anchor_block = get_genesis_forkchoice_store_and_block(spec, state)
+    yield 'anchor_state', state
+    yield 'anchor_block', anchor_block
+    anchor_root = get_anchor_root(spec, state)
+    assert spec.get_head(store) == anchor_root
+    test_steps.append({
+        'checks': {
+            'head': get_formatted_head_output(spec, store),
+        }
+    })
+
+    # Build block that serves as head ONLY on timely arrival, and ONLY in that slot
+    state_1 = genesis_state.copy()
+    next_slots(spec, state_1, 3)
+    block_1 = build_empty_block_for_next_slot(spec, state_1)
+    signed_block_1 = state_transition_and_sign_block(spec, state_1, block_1)
+
+    # Build block that serves as current head, and remains the head after block_1.slot
+    state_2 = genesis_state.copy()
+    next_slots(spec, state_2, 2)
+    block_2 = build_empty_block_for_next_slot(spec, state_2)
+    block_2.body.graffiti = spec.Bytes32(hex(random.getrandbits(8 * 32))[2:].zfill(64))
+    signed_block_2 = state_transition_and_sign_block(spec, state_2.copy(), block_2)
+    while spec.hash_tree_root(block_1) > spec.hash_tree_root(block_2):
+        block_2.body.graffiti = spec.Bytes32(hex(random.getrandbits(8 * 32))[2:].zfill(64))
+        signed_block_2 = state_transition_and_sign_block(spec, state_2.copy(), block_2)
+    assert spec.hash_tree_root(block_1) < spec.hash_tree_root(block_2)
+
+    # Tick to block_1 slot time
+    spec.on_tick(store, store.genesis_time + block_1.slot * spec.config.SECONDS_PER_SLOT)
+
+    # Process block_2
+    yield from tick_and_add_block(spec, store, signed_block_2, test_steps)
+    assert store.proposer_score_boost.root == spec.Root()
+    assert spec.get_head(store) == spec.hash_tree_root(block_2)
+
+    # Process block_1 on timely arrival
+    # The head should temporarily change to block_1
+    yield from tick_and_add_block(spec, store, signed_block_1, test_steps)
+    assert store.proposer_score_boost == spec.LatestMessage(root=spec.hash_tree_root(block_1),
+                                                            epoch=spec.compute_epoch_at_slot(block_1.slot))
```

Comment:
prefer checking each field. same number of lines and reads a bit more direct

```suggestion
    assert store.proposer_score_boost.root == spec.hash_tree_root(block_1)
    assert store.proposer_score_boost.epoch == spec.compute_epoch_at_slot(block_1.slot)
```

Reply:
Actually, now that I think about it... maybe `proposer_score_boost` should just be `proposer_boost_root`. There is zero use of `LatestMessage.epoch` and seems like a bit of an abuse of the semantics of `LatestMessage`